### ISSUE TITLE
question : can this manage selects with multiple options ?

### ISSUE THREAD

**Issue by quasiperfect** [outer-web/filament-settings#1]
hi 

thanks for your work.

can this manage selects with multiple options?

**Comment by SimonBroekaert** [comment]
Hi!

I just tested this out and it works!

For example. If you add this to the settings page schema:

```php

Select::make('general.supported_locales')
  ->label('Locales')
  ->options([
      'en' => 'English',
      'fr' => 'French',
      'de' => 'German',
      'es' => 'Spanish',
      'it' => 'Italian',
      'nl' => 'Dutch',
  ])
  ->multiple(),
```

And you select multiple options, the data will be saved in separate records:
![image](https://github.com/outer-web/filament-settings/assets/35606498/572b9417-925b-4b1b-87a6-e38ae925bbf1)

So if you then use the helper function `setting('general.supported_locales')`, it will return:

```php
[
  'en',
  'fr',
  'nl',
]
```

As you can see, the `setting($key)` helper function will combine db rows together to form an array if they start with the passed $key argument

**Comment by SimonBroekaert** [comment]
Coming back on this, If you update the selected items, the existing items that got deselected will not be deleted.

I will look into this to fix this issue as soon as possible.

**Comment by quasiperfect** [comment]
thanks @SimonBroekaert i will wait for the fix :)

**Comment by SimonBroekaert** [comment]
Hi @quasiperfect ,

This issue is now fixed in **v1.1.0**. The [changelog](https://github.com/outer-web/filament-settings/blob/v1.1.0/CHANGELOG.md) can be found here.

I also took the time to go over all built-in Filament form fields to verify that the data will be created / updated and read out correctly!

Thanks again for posting the initial question which lead to the discovery of this issue.

Please open the issue again if you find any additional problems.

Have a nice day!

**Comment by quasiperfect** [comment]
wow that was fast. 

thanks again for your work.

 i will integrate your plugin with my app and let you know if i find anything

have a great day
